WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/WooShippingSplitShipmentsView.swift

```diff
@@ -61,7 +61,9 @@ struct WooShippingSplitShipmentsView: View {
                     Button(Localization.selectAll) {
                         viewModel.selectAll()
                     }
-                    .disabled(viewModel.isSavingShipmentInfo)
+                    .disabled(
+                        viewModel.currentShipment.isPurchased
+                    )
```

Comment:
Should we keep both conditions? If yes, combining them in a separate property `enablesSelectAll` in the view model would make it clearer.